src/components/editor/editor-pane/tool-bar/table-picker/table-picker.scss

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+
+
+.table-picker-container {
+  z-index: 1111;
+}
+
+body.dark {
+  div.table-cell {
+    @import "../../../../../style/variables.dark";
+    border-top: 1px solid $dark;
+    border-left: 1px solid $dark;
+  }
+
+  div.table-container {
+    @import "../../../../../style/variables.dark";
+    border-bottom: 1px solid $dark;
+    border-right: 1px solid $dark;
+  }
+}
+
+div.table-cell {
+  @import "../../../../../style/variables.light";
+  border-top: 1px solid $dark;
+  border-left: 1px solid $dark;
+}
+
+div.table-container {
+  @import "../../../../../style/variables.light";
+  border-bottom: 1px solid $dark;
+  border-right: 1px solid $dark;
+}
+
+.d-grid {
+  display: grid;
+  grid-template-columns: repeat(10, 15px [col-start]);
+  grid-template-rows: repeat(8, 15px [row-start]);
+}
```

Comment:
`d-grid` is a name pattern that is protected by bootstrap. It should only set the display value and not the columns+rows. You define this rule on root level, so other components could use this.
Because its a rather specific use case I would merge this rule with `table-container`.

Reply:
done